internal/function/uast.go

```diff
@@ -263,6 +289,8 @@ func (u *uastFunc) getUAST(
 		return nil, nil
 	}
 
+	finish(ok)
```

Comment:
what happens if there is an early return?

Reply:
early return happens on error, and here we build uast histogram (measure latency), so if we collect immediately returns on errors then final histogram  may be incorrect. That's why we measure only successfully queries, here.
If we want, we can count errors, but I think it should be done on uast side, but it's up to us.